app/javascript/stylesheets/application.scss

```diff
@@ -300,3 +300,9 @@ html * {
     margin-bottom: 0px;
   }
 }
+
+.single-section-accordion {
+  .govuk-accordion__controls {
+    display: none;
```

Comment:
Hide the 'Open all' button at the top of the govuk accordion, since there's only one section to open.